doc/sphinx-guides/source/developers/making-releases.rst

```diff
@@ -41,20 +47,39 @@ There are a variety of reasons why a step might deserve its own dedicated issue:
 
 Steps don't get their own dedicated issue if it would be confusing to have multiple people involved. Too many cooks in the kitchen, as they say. Also, some steps are so small the overhead of an issue isn't worth it.
 
-Before the release even begins you can coordinate with the project manager about the creation of these issues.
```

Comment:
I took this out and suggested running the script "a week or so before the release" above.